adr/ADR-32.md

```diff
@@ -270,7 +322,12 @@ Note the name of the queue group is fixed to `q` and cannot be changed otherwise
 different implementations on different queue groups will respond to the same
 request.
 
-The handler specified by the client to process requests should operate as any
+For each configured endpoint, a queue subscription should be created.
+
+> Note: Handler subject does not contain the `$SRV` prefix. This prefix
```

Comment:
Not sure why this note is here, perhaps we should clarify that $SRV is a reserved subject group used for monitoring and other generated functionality.

Reply:
Yeah, that was the point, just to make sure we do not add the `$SRV` prefix to endpoint handlers. I added this because I saw this done in other PR so just wanted to clarify.